src/hooks/useSetErc20Balance.ts

```diff
@@ -37,8 +38,9 @@ const SLOT_VALUE_TO_CHECK = 1337_1337_1337_1337_1337_1337_1337_1337_1337n
  */
 export function useSetErc20Balance() {
   const client = useClient()
+  const [loading, setLoading] = useState(false)
```

Comment:
The `useMutation` already returns `isLoading` so we shouldn't need to make any changes here!

Reply:
No problem, updated, it's `isPending` as of v5 as well it looks like:

https://tanstack.com/query/v4/docs/react/reference/useMutation
https://tanstack.com/query/v5/docs/react/guides/mutations